pydantic/main.py

```diff
@@ -1762,7 +1756,7 @@ def create_model(  # noqa: C901
         namespace.update(__validators__)
     namespace.update(fields)
     if __config__:
-        namespace['model_config'] = _config.ConfigWrapper(__config__).config_dict
```

Comment:
Why is this wrapper no longer needed?

Reply:
It is done already in the metaclass, taking into account the config from the bases:

https://github.com/pydantic/pydantic/blob/876bf76f34f9ab313923a3428a4d4aaf00144f43/pydantic/_internal/_model_construction.py#L107-L111